ddl/db_change_test.go

```diff
@@ -791,9 +791,9 @@ func (s *testStateChangeSuite) TestWriteOnlyForDropColumns(c *C) {
 	defer s.se.Execute(context.Background(), "drop table t_drop_columns")
 
 	sqls := make([]sqlWithErr, 2)
-	sqls[0] = sqlWithErr{"update t set c1='5', c3='2020-03-01';", errors.New("[planner:1054]Unknown column 'c3' in 'field list'")}
+	sqls[0] = sqlWithErr{"update t set c1='5', c3='2020-03-01';", errors.New("[planner:1054]Unknown column 'c1' in 'field list'")}
 	sqls[1] = sqlWithErr{"update t t1, t_drop_columns t2 set t1.c1='5', t1.c3='2020-03-01', t2.c1='10' where t1.c4=t2.c4",
-		errors.New("[planner:1054]Unknown column 'c3' in 'field list'")}
+		errors.New("[planner:1054]Unknown column 'c1' in 'field list'")}
```

Comment:
How about the following todo, can this PR solve it?

Reply:
yes, I have fixed the todo